modules/rapid/include/opencv2/rapid.hpp

```diff
@@ -115,10 +117,11 @@ CV_EXPORTS_W void filterCorrespondencies(InputOutputArray pts2d, InputOutputArra
  * @param K camera matrix
  * @param rvec rotation between mesh and camera. Input values are used as an initial solution.
  * @param tvec translation between mesh and camera. Input values are used as an initial solution.
+ * @param cols optional column coordinates in the line bundle
  * @return ratio of search lines that could be extracted and matched
  */
 CV_EXPORTS_W float rapid(InputArray img, int num, int len, InputArray pts3d, InputArray tris, InputArray K,
-                         InputOutputArray rvec, InputOutputArray tvec);
+                         InputOutputArray rvec, InputOutputArray tvec, OutputArray cols = noArray());
```

Comment:
as the line bundle is an implementation detail, I think "coordinates in the line bundle" have no use outside this function and should be hidden, too. I'd rather return the RMSD directly.

Reply:
yeah thats probably right.. OutputArray was just easier to wrap than an output double